scripts/build-post-list.js

```diff
@@ -79,9 +78,8 @@ function walkDirectories(directories, result, sectionWeight = 0, sectionTitle, s
           const fileBaseName = basename(data.slug)  // ex. v2.0.0 | v2.1.0-2021-06-release
           const fileName = fileBaseName.split('-')[0] // v2.0.0 | v2.1.0
 
-          if(fileBaseName.includes('release')) {
+          if(fileBaseName.includes('next-spec') || fileBaseName.includes('next-major-spec')) {
```

Comment:
Hah, yeah, we forgot to update that line when we change the names for release branches 👌🏼